roles/bootstrap-os/tasks/bootstrap-oracle.yml

```diff
@@ -0,0 +1,84 @@
+---
+
+- name: Check presence of fastestmirror.conf
+  stat:
+    path: /etc/yum/pluginconf.d/fastestmirror.conf
+  register: fastestmirror
+
+# fastestmirror plugin actually slows down Ansible deployments
+- name: Disable fastestmirror plugin
+  lineinfile:
+    dest: /etc/yum/pluginconf.d/fastestmirror.conf
+    regexp: "^enabled=.*"
+    line: "enabled=0"
+    state: present
+  when: fastestmirror.stat.exists
+
+- name: Get Oracle Linux Version
+  shell: "cat /etc/oracle-release | grep -oE '[0-9]+' | head -1"
+  register: version_result
+
+- name: Set Fact | Oracle Linux Version
+  set_fact:
+    oracle_version: "{{ version_result.stdout | trim}}"
+
+- name: Check EPEL Installed
```

Comment:
Which packages do you need EPEL for?

Reply:
Gee, I wish I could remember the specific packages this offers that we need. Been too long. I will say, Oracle Linux tends to essentially be a stripped down version of Fedora - so there were probably some essentials that were missing.